stacks/upload-api-stack.js

```diff
@@ -89,6 +116,7 @@ export function UploadApiStack({ stack, app }) {
           PIECE_TABLE_NAME: pieceTable.tableName,
           PIECE_OFFER_QUEUE_URL: pieceOfferQueue.queueUrl,
           FILECOIN_SUBMIT_QUEUE_URL: filecoinSubmitQueue.queueUrl,
+          BLOCKS_CAR_POSITION_TABLE_NAME: blocksCarPositionTable.tableName,
```

Comment:
Also the multihashes queue URL?

Reply:
alright wasn't sure about that one 👍 . adding it